controllers/app.controller.js

```diff
@@ -8,4 +10,14 @@ exports.getTopics = (req, res, next) => {
     .catch((err) => {
         next(err);
     })
+}
+
+exports.getAllEndPoints = (req, res, next) => {
+    fs.readFile('endpoints.json', (err, endpoints) => {
+        if(err) {
```

Comment:
This is inconsistent and errors should be handled using error handling middleware